src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+    private final int random = new Random().nextInt(100);
```

Comment:
now i think all balls were created with the same number
field type should be `Random`, not `int`

also
[All magic numbers in your code should be constants.](https://mate-academy.github.io/jv-program-common-mistakes/java-core/oop/lottery#all-magic-numbers-in-your-code-should-be-constants)
number 100 is a magic number.
it isn't obvious what does it mean, so it is better to give an informative name for this number